src/utils/tableFuntion.test.tsx

```diff
@@ -1,11 +1,46 @@
+/* eslint-disable no-unused-expressions */
+/* eslint-disable import/no-extraneous-dependencies */
+import { describe, expect, it } from 'vitest';
+import { fireEvent, render, screen } from '@testing-library/react';
 import { numberWithCommas } from './functions';
+import { columns } from './tableColumns';
+import { DataGrid } from '../pages/DataGrid';
 
-describe('isBoolean', () => {
-  it('should return the proper separation of numbers in a string', () => {
+describe('Funtions working', () => {
```

Comment:
this test were good why are you changin it?

Reply:
The descriptions and overall writting of the tests were weird, i just changed them so they looked nicer.

Reply:
no need to change them use the config that is already config